test/e2e/watchcache/watchcache_enabled_test.go

```diff
@@ -272,27 +272,30 @@ func testDynamicDiscoverySharedInformerFactory(ctx context.Context, t *testing.T
 	t.Log("Starting DynamicDiscoverySharedInformerFactory")
 	go ddsif.StartWorker(ctx)
```

Comment:
since we use `ddsif.Start(nil)` do we still need to call `StartWorker` ?

Reply:
otherwise  it looks like the `StartWorker` method will simply start the informer as well

Reply:
It turns out we do, because when StartWorker terminates, it stops all started informers.